wifiphisher/common/phishingpage.py

```diff
@@ -336,8 +363,14 @@ def find_user_templates(self):
             # check to see if it is a directory and not in the database
             if (os.path.isdir(os.path.join(self._template_directory, name)) and
                     name not in self._templates):
-                # add it to the list
-                local_templates.append(name)
+				# check for template
+				if (self.search_template(name) == True):
+					# add it to the list
+					local_templates.append(name)
+					print "Found template in: " + name
+				else:
+					# or not
+					print "No template found in: " + name
```

Comment:
can you follow the print style in `pywifiphisher.py`? something like the following:
`[!] No template found in: ****`

Reply:
Yep, of course. Should i include constants for colorful output?